tox.ini

```diff
@@ -100,6 +108,7 @@ commands_pre =
   pip install -e {toxinidir}/ext/opentelemetry-ext-wsgi
   pip install -e {toxinidir}/examples/opentelemetry-example-app
   pip install -e {toxinidir}/ext/opentelemetry-ext-opentracing-shim
+  #pip install -r {toxinidir}/ext/opentelemetry-ext-opentracing-shim/tests/testbed/requirements.txt
```

Comment:
Why comment this out?

Reply:
ops, part of something I was trying out and then realized it was not needed. Will remove it.